src/lib/cli_lib/commands.ml

```diff
@@ -191,6 +191,9 @@ module Vrf = struct
       Exceptions.handle_nicely
       @@ fun () ->
       let env = Secrets.Keypair.env in
+      let constraint_constants =
+        Genesis_constants.Compiled.constraint_constants
```

Comment:
Isn't this moving in the wrong direction?

Reply:
My thought process was the same in all of the `command` definitions across the PR: You are already accessing command line arguments at this point, and the CLI options are how you will determine the config. This could be in one of several ways, e.g. pointing you to a file or directory, overriding values in some way, etc. 

If you pass these as an argument to this function as it was before, I guess you could define this as the `command_group` level, but not every command requires the same configuration arguments